changelog.txt

```diff
@@ -1,3 +1,30 @@
+14.0 (2024-04-13)
+------------------------------------------------------------------------
+Update: New title game for 14.0
+Fix #12477: Use std::filesystem::rename instead of Windows Shell API call (#12478)
+Fix #12233: Mini order list overlaps vehicle group name (#12423)
+Fix #12114: Viewport coords of crashed aircraft not updated when falling (#12424)
+Fix #12395: Ensure president name widget is tall enough (#12419)
+Fix #12415: Aircraft::tile is valid only for front vehicle (#12416)
+Fix #12387: [NewGRF] Wrong tile offset passed to rail station CB 149 (slope check)
+Fix #12388: Vehicle::CopyVehicleConfigAndStatistics not releasing unit number (#12389)
```

Comment:
```suggestion
Fix #12388: Autoreplacing train heads slowly made the unit number grow (#12389)
```

Or something. Not sure. For sure not what is in there now :) Otherwise we can just leave it out, honestly.